src/pages/Home/index.js

```diff
@@ -1,15 +1,527 @@
-import React, { Fragment } from 'react';
-
-import SiteTitle from 'components/SiteTitle';
-import SiteDetails from 'components/SiteDetails';
-import UsernameForm from 'components/UsernameForm';
-
-const Home = () => (
-  <Fragment>
-    <SiteTitle>Frogtoberfest Checker</SiteTitle>
-    <UsernameForm />
-    <SiteDetails />
-  </Fragment>
-);
+import React,{ Fragment, useEffect } from 'react';
+import { Link } from 'react-router-dom/cjs/react-router-dom.min';
+
+
+import Icon1 from 'assets/images/icon-1.svg';
+import Icon2 from 'assets/images/icon-2.svg';
+import Icon3 from 'assets/images/icon-3.svg';
+import Icon4 from 'assets/images/icon-4.svg';
+import Icon5 from 'assets/images/icon-5.svg';
+import Icon6 from 'assets/images/icon-6.svg';
+import Icon7 from 'assets/images/icon-7.svg';
+import Img1 from 'assets/images/img-1.png';
+import Img2 from 'assets/images/img-2.png';
+import Img3 from 'assets/images/img-3.png';
+import Img4 from 'assets/images/img-4.png';
+import Img5 from 'assets/images/img-5.png';
+import Img6 from 'assets/images/img-6.png';
+import Img7 from 'assets/images/img-7.png';
+import Img8 from 'assets/images/img-8.png';
+import Img9 from 'assets/images/img-9.png';
+import Antenna from 'assets/images/antenna.png';
+import Logo from 'assets/images/logo.svg';
+import SiteHeader from 'components/SiteHeader';
+
+const Home = () => {
+  const lastDate = new Date('2023-10-01');
+  const todayDate = new Date();
+  const dif = Math.abs(lastDate - todayDate) / 1000;
+
+  const countDownClock = (number = 100, format = 'seconds') => {
+    const d = document;
+    const days1Element = d.querySelector('#day-1');
+    const days2Element = d.querySelector('#day-2');
+    const hour1Element = d.querySelector('#hour-1');
+    const hour2Element = d.querySelector('#hour-2');
+    const minute1Element = d.querySelector('#minute-1');
+    const minute2Element = d.querySelector('#minute-2');
+    let countdown;
+
+    convertFormat(format);
+
+    function convertFormat(format) {
+      switch (format) {
+        case 'seconds':
+          return timer(number);
+        case 'minutes':
+          return timer(number * 60);
+        case 'hours':
+          return timer(number * 60 * 60);
+        case 'days':
+          return timer(number * 60 * 60 * 24);
+      }
+    }
+
+    function timer(seconds) {
+      const now = Date.now();
+      const then = now + seconds * 1000;
+
+      // For first render
+      const secondsLeft = Math.round((then - Date.now()) / 1000);
+
+      displayTimeLeft(secondsLeft);
+
+      countdown = setInterval(() => {
+        const secondsLeft = Math.round((then - Date.now()) / 1000);
+
+        if (secondsLeft <= 0) {
+          clearInterval(countdown);
+
+          return;
+        }
+
+        displayTimeLeft(secondsLeft);
+      }, 1000);
+    }
+
+    function displayTimeLeft(seconds) {
+      const totalDaysRemaining = Math.floor(seconds / 86400);
```

Comment:
can we move the numbers like 86400 and 3600 to a suitable constant like 30_DAYS, etc.